error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -805,6 +809,20 @@ Flux<T> after(Flux<? extends Iterable<T>> flux, int prefetch) {
     }
   }
 
+  /** Prefer {@link Flux#count()} over collecting into a list and counting its elements. */
+  static final class FluxCount<T> {
+    @BeforeTemplate
+    Mono<Integer> before(Flux<T> flux) {
+      return flux.collect(toImmutableList())
+          .map(Refaster.anyOf(Collection::size, ImmutableList::size, List::size));
```

Comment:
There's also `ImmutableCollection::size` :smile:

Reply:
Of course! 
Thanks :)